assessee/tests.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+from users.models import OdiUser
+import json
+
+
+class ViewsTestCase(TestCase):
+    def setUp(self):
+        self.url = reverse("assessee_dashboard")    # use the view url
```

Comment:
Such an elegant way to get the URL. I will modify my part of the application to also use the reverse function rather than defining a constant.